src/components/pages/blog/posts-board/community-links/community-links.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+import Container from 'components/shared/container';
+import Heading from 'components/shared/heading';
+import Link from 'components/shared/link';
+import GithubIcon from 'icons/github-dark.inline.svg';
+import SlackIcon from 'icons/slack-dark.inline.svg';
+import TwitterIcon from 'icons/twitter-dark.inline.svg';
+
+const title = 'Community';
+const items = [
+  {
+    icon: SlackIcon,
+    logoName: 'Slack',
+    title: 'Slack',
+    description:
+      'For live conversation and quick questions, join the Cilium Slack workspace. Don’t forget to say hi!',
+    linkText: 'Join Slack workspace',
+    url: 'https://cilium.herokuapp.com/',
+    target: '_blank',
+  },
+  {
+    icon: TwitterIcon,
+    logoName: 'Twitter',
+    title: 'Twitter',
+    description: 'Don’t forget to follow Cilium on Twitter for the latest news and announcements.',
+    linkText: 'follow Cilium on Twitter',
+    url: 'https://twitter.com/ciliumproject',
+    target: '_blank',
+  },
+  {
+    icon: GithubIcon,
+    logoName: 'GitHub',
+    title: 'GitHub',
+    description: 'Cilium uses GitHub tags to maintain a list of asked questions.',
+    linkText: 'Join Github',
+    url: 'https://github.com/cilium/cilium',
+    target: '_blank',
+  },
+];
+
+const Community = () => (
+  <section className="overflow-hidden bg-white py-8 sm:overflow-visible md:py-10 lg:py-14">
+    <Container>
+      <Heading className="text-center" tag="h2">
+        {title}
+      </Heading>
+      <div className="mt-7 flex flex-col justify-between space-y-5 md:mt-10 md:flex-row md:space-y-0 md:space-x-4 lg:mt-14 xl:space-x-8">
+        {items.map(({ icon: Icon, logoName, title, description, linkText, url, target }, index) => (
+          <Link
+            to={url}
+            target={target}
```

Comment:
Nitpick: I could have added the attribute `rel` with `target="_blank"`.
```suggestion
            target={target}
            rel={target ? 'noopener noreferrer' : null}
```